net/xia/ppal_ether/main.c

```diff
@@ -168,6 +169,316 @@ static void local_free_ether(struct fib_xid_table *xtbl, struct fib_xid *fxid)
 	kfree(leid);
 }
 
+/* Main-FXID. */
+struct fib_xid_ether_main {
+	struct xip_dst_anchor   xem_anchor;
+	struct net_device       *host_interface;
+	struct list_head        interface_common_addr;
+	/* Hardware address of the neighbor. */
+	u8                      ha[ALIGN(MAX_ADDR_LEN, sizeof(long))];
```

Comment:
I think letting it be there might be beneficial for NWP in the case of a neighbour MAC address change rather than deleting an entry and then creating a new one. If you still want me to remove this, i will do so.